### DISCUSSION TITLE
Playtime: Ctrl+mouseclick and Shift+mouseclick multiple selection of clips and copy paste dragging to other rows/columns

### DISCUSSION THREAD

**Discussion by joungle123** [helgoboss/helgobox#1251] [Ideas]
Ctrl+mouseclick and Shift+mouseclick does not select multiple items at once, it would be great if they could select multiple line items (clips) in player and be able to drag and drop them to a new line (dragging, copy pasting)

**Comment by helgoboss** [comment]
This is a duplicate of #1045. You can comment there if you have specific needs. Closing this.
